### PR TITLE
Create visualizationkerascv.py

### PR DESCRIPTION
**Pull request by ushareng**
Discussion of visualization offerings from KerasCV

Colab Notebook: https://colab.research.google.com/drive/1dFUekCag4s4Z94GTwnPVEv295jGLK3yp?usp=sharing

@martin-gorner @ianstenbit

### CODE REVIEW

**Comment by sachinprasadhs** [comment]
Could you please follow the link https://cla.developers.google.com/. and sign a Contributor License Agreement (CLA)

**Comment by ushareng** [comment]
> Could you please follow the link https://cla.developers.google.com/. and sign a Contributor License Agreement (CLA)

image

I signed the agreement yesterday .

**Comment by haifeng-jin** [comment]
@sachinprasadhs Seems the CLA still doesn't pass. Would you help guide the contributor to sign it? Thanks!

**Comment by haifeng-jin** [comment]
@ianstenbit Would you please help take a look at this PR? Thanks!

File: examples/vision/visualizationkerascv.py [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    cols=2,
+    y_true=bounding_boxes,
+    scale=5,
+    font_scale=0.7,
+    bounding_box_format="xywh",
+    class_mapping=class_mapping,
+)
+
+"""
+## KerasCV visualisation for verifying augementation
```
Comment (ianstenbit) [inline examples/vision/visualizationkerascv.py:183]: s/augementation/augmentation

**Review by ianstenbit — COMMENTED** [review]
Thanks for the PR!

My overall take is that it's a bit long for what it covers.

I think ideally we would really succinctly show off how to use our API with a batch of images (maybe even just loaded from Imgur or some local files), and all the heavyweight dataloading, etc. is a bit distracting.

One option is to follow what we do with our visualization examples (e.g. [this one](https://github.com/keras-team/keras-cv/blob/master/examples/layers/preprocessing/segmentation/random_shear_demo.py)).

I think we basically just want to show:

- A batch of images
- A batch of images with predicted + GT boxes
- A batch of images with predicted + GT masks

**Pull request closed without merging**